doc/help.rst

```diff
@@ -2,38 +2,98 @@
 Help
 ####
 
-You can use the **Help** pane to find, render and display rich documentation for any object with a docstring, including modules, classes, functions and methods.
+You can use the **Help** pane to find, render and display rich documentation for any object with a docstring, including modules, classes, functions and methods. This allows you to access documentation easily directly from Spyder not having to break your workflow.
+
+|
+
+.. raw:: html
+
+	<iframe src="https://giphy.com/embed/H1quFSJ3LSwmVwvIhU" width="480" height="300" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/H1quFSJ3LSwmVwvIhU">via GIPHY</a></p>
+
+
+|
+
+
+You can also access Spyder's tutorial from here, which will guide you through some basic steps for using its key features.
+
+.. image:: images/help/help-tutorial.png
+   :width: 600
+   :align: center
+   :alt: Spyder Help pane showing the link for the Spyder's tutorial
+
+|
```

Comment:
As mentioned before this is information sparse enough that if it should still be included, it should be half-width, right-aligned (or left if you really want to switch everything, but I had some reason for preferring right) if we include it at all, and with text flowing around it like images elsewhere of that size in the style guide.